docs/guides/documentdb/autoscaler/storage/index.md

```
@@ -0,0 +1,350 @@
+title: DocumentDB Storage Autoscaling
+menu:
+  docs_{{ .version }}:
+    identifier: dc-auto-storage
+    name: Storage Autoscaling
+    parent: dc-auto-scaling
+    weight: 30
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)